guides/common/modules/proc_rex-pull-based-transport-tuning.adoc

```diff
@@ -0,0 +1,33 @@
+[id="REX_pull_based_transport_tuning_{context}"]
+= REX pull-based transport tuning
+
+{Project} has a pull-based transport that has the features of REX but with a deployment model similar to katello-agent. 
+This transport uses MQTT as its messaging protocol and includes an MQTT client running on each host.
+In order to tune the MQTT server (`mosquitto`) and increase the number of hosts connected to it, you can follow the following procedure on the {ProjectServer}:
+
+.Procedure
+
+. Enable pull-based remote execution on your {ProjectServer} or {SmartProxyServer}:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+{foreman-installer} --foreman-proxy-plugin-remote-execution-script-mode pull-mqtt
```

Comment:
This is identical to the file above. What's the reasoning? Do you want to show both increasing and decreasing?